assets/js/blocks/order-confirmation/status/style.scss

```
@@ -0,0 +1,16 @@
+.wc-block-order-confirmation-status {
+	margin: $gap 0;
+	p {
+		margin: 0;
+	}
+	.wc-block-order-confirmation-status__actions {
+		margin-top: $gap;
+
+		a {
+			margin-right: $gap;
+		}
+		a.has-link-color {
+			color: inherit;
+		}
```

Shouldn't we fully utilise SCSS in the following case:

```suggestion
		a {
			margin-right: $gap;		
			
			&.has-link-color {
				color: inherit;
			}
		}
```